docker/fetch_tezos_sources.py

```diff
@@ -24,8 +24,6 @@
     ]
 )
 
```

Comment:
We should really try to not forget this in the future.
```suggestion
# NOTE: it's important to keep the `tezos/.git` directory here, because the
# git tag is used to set the version in the Octez binaries.
```